src/component/Form.js

```diff
@@ -0,0 +1,22 @@
+const Form = () => (
+
+  <form className="form-container">
+    <p>ADD NEW BOOK</p>
+    <div className="form-wrap d-flex">
+      <input
+        className="input-title"
+        type="text"
+        placeholder="Book title"
+        name="title"
+      />
+
+      <select defaultValue="Author" className="input-Author" name="Author">
+        <option value="" selected disabled>Author</option>
+      </select>
```

Comment:
- [ ] Kindly make this an input field as the user will have to type the author of the book instead of selecting from a dropdown. The authors of the books to be created are not pre-defined nor known before a book is created for an author and hence, the user has to type the name of the author. In that case, please make this an input field, not a select element. 👍